src/helpers/utils.tsx

```diff
@@ -458,15 +458,15 @@ export function getColor(taskObject: InitialTask): Color {
   }
 }
 
-/** filters out plans whose jurisdiction id is null
+/** filters out plans whose jurisdiction id is null or has a value of 'null'
  * @param {Plan []} plansArray - a list of pre-filtered plans
  *
  * @return {Plan []} - A list of filtered plans
  */
 export const removeNullJurisdictionPlans = (plansArray: Plan[]): Plan[] => {
   return plansArray.filter((plan: Plan) => {
-    const jurisdictionID = plan.jurisdiction_id.toLowerCase().trim();
-    return !jurisdictionID.includes('null');
+    const jurisdictionId = plan.jurisdiction_id?.toLowerCase().trim();
+    return !!jurisdictionId && !jurisdictionId.includes('null');
```

Comment:
Is it completely safe to remove `.trim()`?

Reply:
I don't think so. If I remember correctly the values that were originally handled by this looked sth like ~  '  NULL'

Reply:
yeah I recall that as well.  Should we add trim back to this code and ensure we have a test that covers the `' NULL'` cases?

Reply:
The existing function [test](https://github.com/onaio/reveal-frontend/blob/4b8394d538e2cb9c7fe77e792e32849792ea67ef/src/helpers/tests/utils.test.tsx#L326-L330) covers `'  NULL'`  cases ([fixtures](https://github.com/onaio/reveal-frontend/blob/4b8394d538e2cb9c7fe77e792e32849792ea67ef/src/store/ducks/tests/fixtures.ts#L187-L217)).